docs/content-services/components/search-date-range-advanced-tabbed.component.md

```diff
@@ -0,0 +1,103 @@
+---
+Title: Search date range advanced component
+Added: v6.1.0
```

Comment:
Version seems incorrect. We have now 6.2.0

Reply:
Updated version to 6.2.0 and Last Reviewed date to 2023-06-28